pkg/operator/ceph/cluster/mgr/dashboard_test.go

```diff
@@ -157,3 +157,30 @@ func TestStartSecureDashboard(t *testing.T) {
 	assert.True(t, kerrors.IsNotFound(err))
 	assert.Nil(t, svc)
 }
+
+func TestCephVersionsNewCephDashboardCommand(t *testing.T) {
```

Comment:
What about older versions of octopus? I don't see that test.

Reply:
I reffered [this](https://github.com/rook/rook/issues/7498#issuecomment-810267610) comment for versions.